src/EnergyPlus/OutputProcessor.cc

```diff
@@ -239,8 +239,8 @@ namespace OutputProcessor {
     }
 
     void SetupTimePointers(EnergyPlusData &state,
-                           std::string const &TimeStepTypeKey, // Which timestep is being set up, 'Zone'=1, 'HVAC'=2
-                           Real64 &TimeStep                    // The timestep variable.  Used to get the address
+                           OutputProcessor::SOVTimeStepType const &TimeStepTypeKey, // Which timestep is being set up, 'Zone'=1, 'HVAC'=2
```

Comment:
SetupTimePointers now takes an enum instead of a string, modified the fatal error message below to use the constexpr string_view worker function.